app/scripts/react-directives/components/modals/modals.jsx

```
@@ -31,20 +31,22 @@
         }
       };
-      document.addEventListener('keydown', handleEsc, true);
-      document.addEventListener('mousedown', handleClick, true);
-      document.addEventListener('touchstart', handleClick, true);
-      var backdrop = document.createElement('div');
+      if (onCancel) {
+        document.addEventListener('keydown', handleEsc, true);
+        document.addEventListener('mousedown', handleClick, true);
+        document.addEventListener('touchstart', handleClick, true);
+      }
+      let backdrop = document.createElement('div');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'backdrop' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)